centos_ci.sh

```diff
@@ -308,6 +308,14 @@ function prepare_for_proxy() {
   firewall-cmd --zone=public --add-port=$INTEGRATION_PROXY_CUSTOM_PORT/tcp;
 }
 
+function perform_nightly() {
+  setup_kvm_docker_machine_driver;
+  cd $GOPATH/src/github.com/minishift/minishift
+  make prerelease synopsis_docs link_check_docs
+  MINISHIFT_VM_DRIVER=kvm make integration_all GODOG_OPTS=""
```

Comment:
Is this passing? I am suspicious of coolstore.feature, which was not fixed yet. And also of timeout, which is 3600s per default (https://github.com/minishift/minishift/blob/master/Makefile#L35). I would suggest to increase it so we will not hit the limit on next additions to integration test suite. This would be done by adding: `TIMEOUT=7200s` for example. More here: https://docs.openshift.org/latest/minishift/contributing/developing.html#additional-parameters.

Reply:
Even i am surprised how the it CI passed with the command ```MINISHIFT_VM_DRIVER=kvm make integration_all GODOG_OPTS=""```

>I would suggest to increase it so we will not hit the limit on next additions to integration test suite. This would be done by adding: TIMEOUT=7200s

@budhrg For running integration_all test what is the time taken by CI ?

Reply:
@amitkrout , not known yet.